testing/test-vectors/prng-tests.lisp

```diff
@@ -2,5 +2,8 @@
 (in-package :crypto-tests)
 
 (rtest:deftest :prng-fortuna (run-test-vector-file :prng *prng-tests*) t)
+(rtest:deftest :prng-fortuna-urandom (let ((prng (crypto:make-prng :fortuna :seed urandom)))
```

Comment:
I think I figured out how to make test runs actually fail, but some existing tests fail: https://travis-ci.org/froydnj/ironclad/jobs/225603790